src/arvgvdevice.c

```diff
@@ -726,6 +723,11 @@ auto_packet_size (ArvGvDevice *gv_device, gboolean exit_early, GError **error)
 		guint current_size = packet_size;
 
 		do {
+			if (g_timer_elapsed (timer, NULL) > 1.0) {
```

Comment:
We could also make this timeout configurable.
I was just not sure:
1) Whether we want this at all.
2) Whether we want to change the APIs of `arv_gv_device_auto_packet_size`/`arv_camera_gv_auto_packet_size` for this or
3) Want to introduce new methods, e.g. `arv_gv_device_auto_packet_size_timeout`/`arv_camera_gv_auto_packet_size_timeout`